geomstats/information_geometry/dirichlet.py

```diff
@@ -487,354 +401,16 @@ def exp(self, tangent_vec, base_point, n_steps=N_STEPS):
             Tangent vector at base point.
         base_point : array-like, shape=[..., dim]
             Base point.
-        n_steps : int
-            Number of steps for integration.
-            Optional, default: 100.
 
         Returns
         -------
         exp : array-like, shape=[..., dim]
             End point of the geodesic starting at base_point with
             initial velocity tangent_vec and stopping at time 1.
         """
-        stop_time = 1.0
-        geodesic = self._geodesic_ivp(base_point, tangent_vec, n_steps)
-        exp = geodesic(stop_time)
-
-        return exp
-
-    def _approx_geodesic_bvp(
```

Comment:
I've deleted it because with the solvers we have no way of choosing to use this function. I'm aware we still need to bring this strategy for initialization. The code is still available in history if we delete it.

With the default solvers it seems to work pretty well...

Do you prefer to keep it or looking later to the history is an option?

Reply:
I would prefer keeping it for now, but I agree it should be temporary.